bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/util/DataWrapperUtil.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.util;
+
+import com.google.bigtable.v2.Cell;
+import com.google.bigtable.v2.CheckAndMutateRowRequest;
+import com.google.bigtable.v2.CheckAndMutateRowResponse;
+import com.google.bigtable.v2.Column;
+import com.google.bigtable.v2.Family;
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowCell;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.ByteString;
+
+/**
+ * This class has utility methods for model conversion for
+ * {@link com.google.cloud.bigtable.core.IBigtableDataClient}.
+ */
+public class DataWrapperUtil {
```

Comment:
Should I change the name of utility? As it could be used for BigtableTableAdminClient as well

Reply:
why do you think you need this wrapper? you can have this method in same wrapper impl class. Is it going to be used from anywhere else I dont think so.

Reply:
Yes, it is not being used anywhere else, I will move it back in the wrapper Impl file.